cpp/locality/locality-files.cmake

```
@@ -0,0 +1,39 @@
+set(VOROPP_SOURCE_DIR "${PROJECT_SOURCE_DIR}/extern/voro++/src")
+
+set(locality_sources
+    locality/AABBQuery.cc
+    locality/FilterRAD.cc
+    locality/FilterSANN.cc
+    locality/LinkCell.cc
+    locality/NeighborComputeFunctional.cc
+    locality/NeighborList.cc
+    locality/PeriodicBuffer.cc
+    locality/Voronoi.cc
+    # For now, compile voro++ object in directly.
```

Is there a problem with making this a permanent solution?
```suggestion
    # Compile voro++ object in directly.
```

---

We can do it permanently. I think this comment was added a long time ago.